docs/blog/newsletter-november-2020.rst

```diff
@@ -0,0 +1,141 @@
+
+.. post:: November 02, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2020
+#########################################
+
+Hello friends - it’s Beth and the newsletter team here.
+
+And just like that, Prague has been and gone. I spent the days after conference feeling a strange mix of revived and exhausted - in other words, just how I always feel after a Write the Docs conference. Not a bad outcome given the move online! I’m glad it turns out we can still gather and be supported and inspired, even when pandemics conspire against us. Plus, videos of both the main stage talks and lightning talks are now up on `our YouTube playlist <https://www.youtube.com/playlist?list=PLZAeFn6dfHpmRWZJaUwQzsdagW2TtRI2x>`__ - enjoy!
```

Comment:
```suggestion
And just like that, WTD Prague has been and gone. I spent the days after conference feeling a strange mix of revived and exhausted - in other words, just how I always feel after a Write the Docs conference. Not a bad outcome given the move online! I’m glad it turns out we can still gather and be supported and inspired, even when pandemics conspire against us. Plus, videos of both the main stage talks and lightning talks are now up on `our YouTube playlist <https://www.youtube.com/playlist?list=PLZAeFn6dfHpmRWZJaUwQzsdagW2TtRI2x>`__ - enjoy!
```

Reply:
FWIW this is very you-centric, given that we did the same with Portland ;-)

Reply:
That last "Plus," is a little dangling as well

Reply:
Maybe "the Prague conference"?

Reply:
It is very me-centric. I'll tone it down a bit.